cmake/CompilerOptions.cmake

```diff
@@ -6,12 +6,6 @@ include(CMakePushCheckState)
 set(CXXOPT_DEBUG "-g")
 set(CXXOPT_CXX11 "-std=c++11")
 
-if(CMAKE_CXX_COMPILER_ID MATCHES "XL")
```

Comment:
Is this no longer an issue?

Reply:
We stopped support for the IBM XL compiler long ago, so those flags will be the least of your concerns if you try to compile with it today. I just did that clean up when I was modifying something else in the same file.